projects/pastanaga/src/lib/textfield/select.component.html

```diff
@@ -1,6 +1,9 @@
 <div class="pa-field pa-field-select" [class.pa-field-error]="hasError || !!errorMessage">
     <select class="pa-field-control" [id]="id" [attr.disabled]="disabled ? true : null"
+            [class.pa-field-control-lessen]="isLessen"
```

Comment:
are you sure the style is correct when readonly and lessen are both `true`? 
in input, `[class.pa-field-control-lessen]="isLessen && !isReadOnly"` is because `pa-field-control-lessen` was interfering with `pa-field-control-readonly-lessen` class.